src/raft/raft_log.rs

```diff
@@ -53,22 +53,25 @@ pub struct RaftLog<T: Storage> {
     // been instructed to apply to its state machine.
     // Invariant: applied <= committed
     pub applied: u64,
+
+    pub tag: String,
 }
 
 impl<T> ToString for RaftLog<T>
     where T: Storage
 {
     fn to_string(&self) -> String {
-        format!("committed={}, applied={}, unstable.offset={}, unstable.entries.len()={}",
+        format!("committed={}, applied={}, unstable.offset={}, unstable.entries.len()={}, tag={}",
```

Comment:
I think we can add tag here https://github.com/pingcap/tikv/blob/e38392f3fe82c1b3721aefdb385d42a9d16cef97/src/raft/raft_log.rs#L398